packages/vaadin-grid/theme/lumo/vaadin-grid-sorter-styles.js

```diff
@@ -30,6 +33,11 @@ registerStyles(
       color: var(--lumo-primary-text-color);
     }
 
+    :host(:hover) [part='indicators'] {
+      color: var(--lumo-body-text-color);
+      opacity: 0.6;
+    }
```

Comment:
Added another indication of interactivity, since the cursor changes from `pointer` (defined in "core" styles) to `default`.